index.js

```diff
@@ -1,26 +1,36 @@
-var path        = require('path')
-var loaderUtils = require('loader-utils')
+var path = require('path');
+var glob = require('glob');
+var loaderUtils = require('loader-utils');
 
-module.exports = function(source) {
+module.exports = function (source) {
+    var query = loaderUtils.parseQuery(this.query);
+    var manifest = query.manifest || (query.manifestPath ? requireManifests(query.manifestPath) : false);
 
-  var query = loaderUtils.parseQuery(this.query)
-  var manifest = query.manifest || (query.manifestPath ? requireManifest(query.manifestPath) : false)
-
-  if (manifest) {
-    for ( var key in manifest ) {
-      var value = manifest[key]
-      var pattern = new RegExp(key, 'gm')
-      source = source.replace(pattern, value)
+    if (manifest) {
+        for (var key in manifest) {
+            var value = manifest[key];
+            var pattern = new RegExp(key, 'gm');
+            source = source.replace(pattern, value);
+        }
     }
-  }
 
-  return source
+    return source;
+};
+
+function requireManifests(manifestGlob) {
+    return glob.sync(manifestGlob).map(function (path) {
+        return requireManifest(path);
+    }).filter(function (manifest) {
+        return manifest !== false;
+    }).reduce(function (manifest, current) {
+        return Object.assign(manifest, current);
+    }, {});
 }
 
 function requireManifest(manifestPath) {
-  try {
-    return require(manifestPath)
-  } catch(error) {
-    return false
-  }
+    try {
+        return require(process.cwd() + path.sep + manifestPath);
```

Comment:
I might use `path.resolve()` here, instead of string concat and path.sep.

Reply:
👍, good catch. I realize I haven't even looked at this PR 🙈

Reply:
Fun fact though: I didn't know `path.sep` was a thing.

Reply:
@greypants funny thing is I usually use `path.resolve()`, hence I have no idea what happened in my brain here ;)